src/CoreBundle/CronJob/CronJobScheduler.php

```diff
@@ -103,6 +103,17 @@ public function calculateCurrentPlannedExecutionDate(CronJobScheduleDataModel $s
         return $plannedExecution;
     }
 
+    private function isLess(\DateInterval $a, \DateInterval  $b): bool
+    {
+        $now = $this->timeProvider->getDateTime();
+        $aNow = clone $now;
+        $aNow = $aNow->add($a);
+        $bNow = clone $now;
+        $bNow->add($b);
+
+        return ($aNow >= $bNow);
```

Comment:
Shouldn't it be < ?

Reply:
the `<` was correct, but the entire method has been removed anyway